guides/Getting-Started.md

```diff
@@ -16,48 +16,48 @@ The Purescript compiler (`purs`) can be installed with npm:
 
 #### Setting up the Development Environment
 
-PureScript's core libraries are configured to use the [Pulp](https://github.com/purescript-contrib/pulp) build tool, and packages are available in the [Bower registry](http://bower.io/search/?q=purescript-).
+PureScript's core libraries are configured to use the [Spago](https://github.com/spacchetti/spago) package manager and build tool.
 
-If you don't have Pulp and Bower installed, install them now:
+If you don't have Spago installed, install it now:
 
-    npm install -g pulp bower
+    npm install -g spago
 
-Create a new project in an empty directory using `pulp init`:
+Create a new project in an empty directory using `spago init`:
 
-    pulp init
+    spago init
 
 Your directory should now contain the following files:
 
-- `bower.json` - contains library dependency information
-- `bower_components/` - a directory for installed dependencies
+- `packages.dhall` - contains Spago configuration
+- `spago.dhall` - contains library dependency information
 - `src/Main.purs` - Entry point module for your project
 - `test/Main.purs` - An empty test suite
 
 At this point, you should be able to build the project and run the tests:
 
-    pulp build
-    pulp test
+    spago build
+    spago test
 
 You should see output similar to the following:
 
-    * Building project in /Users/paf31/Documents/Code/purescript/pulp-test
+    * Building project in /Users/paf31/Documents/Code/purescript/spago-test
     * Build successful. Running tests...
     You should add some tests.
     * Tests OK.
-    
+
 If everything was built successfully, and the tests ran without problems, then the last line should state "Tests OK".
 
 #### Installing Dependencies
 
-Dependencies can be installed using Bower. We will be using the `purescript-lists` library shortly, so install it now:
+Dependencies can be installed using Spago. We will be using the `purescript-lists` library shortly, so install it now:
```

Comment:
Is it worth adding a line here that by convention / for historical reasons PureScript packages have a `purescript-` prefix, but when installing them you just use the package name? 

I can see it being a little odd to instruct someone to install `purescript-lists` by typing `spago install lists` without them knowing whether that's how all packages work or just this one.

Fortunately I believe Spago will install `purescript-lists` by trimming the `purescript-` prefix.

Reply:
> Fortunately I believe Spago will install `purescript-lists` by trimming the `purescript-` prefix.

This is right. I think it's ok to reference the package with the prefix, as that's the name of the repo anyways? Though I have no strong opinion on this :man_shrugging: